### DISCUSSION TITLE
How to disable Zed autocomplete suggestions in comments?

### DISCUSSION THREAD

**Discussion by soham-suki** [zed-industries/zed#13339] [Feature Requests]
When writing comments, I often use periods to end sentences (and hit enter to go to the next line)
However, Zed suggests completions when I hit periods in comments:
![image](https://github.com/zed-industries/zed/assets/171621461/5dabbb3c-7637-41c7-99df-1acc62bf8df9)
Then when I hit enter, it thinks I wanted to use its suggestions, so it adds that after the period. To prevent this, I have to constantly hit Esc to dismiss suggestions.

Is there a way to disable suggestions inside of comments?

**Comment by soham-suki** [comment]
Bump!

**Comment by soham-suki** [comment]
Bump!

**Comment by soham-suki** [comment]
Bump!

**Comment by soham-suki** [comment]
Bump!

**Comment by soham-suki** [comment]
Created an issue out of this discussion: #14372

**Comment by crabmusket** [comment]
Still affecting regular LSP suggestions, not edit predictions!
